### ISSUE TITLE
[A11y] Screen reader cannot read the contents of desktop client

### ISSUE THREAD

**Issue by saw-jan** [opencloud-eu/desktop#860]
## Steps to reproduce
1. Open desktop client
2. Open screen reader. E.g: orca
3. Focus the desktop client window
4. Nothing from the screen reader ⚠️ 

#### Findings
The problem seems to be how the client is built. The client is built using the craft build system which uses the qtbase library from the kde cache. The cached qtbase doesn't include the necessary a11y implementations/modules.

The fix would be to build the client with qtbase that has necessary a11y modules. When using craft, we need to build the `libs/qt6/qtbase` from source instead of using from cache. And while build the qtbase, make sure to confirm that `ATSPI` is found and `Accessibility` feature says `yes` 
```
Qt Gui:
  Accessibility .......................... yes
```
check the build:
```
strings ./main/linux-gcc-x86_64/lib/libQt6Gui.so | grep -i QSpiAccessibleBridge
```

## Expected behavior
Screen reader reads the contents of desktop client window

## Actual behavior
Screen reader is silent when focusing the desktop client window

## Setup
- Your operating system: `Ubuntu 24.04`
- Client version:
```
OpenCloud 3.0.3.2073 [72fe8f]
Libraries Qt 6.9.2, OpenSSL 3.5.4 30 Sep 2025
Using virtual files plugin: off
OS: ubuntu-6.8.0-106-generic (build arch: x86_64, CPU arch: x86_64)
QPA: xcb
```

**Comment by saw-jan** [comment]
Same as https://github.com/opencloud-eu/desktop/issues/821
